v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/cassandra/mappings/CassandraMappingsProvider.java

```diff
@@ -84,38 +92,121 @@ public class CassandraMappingsProvider {
 
   private static final CassandraMappings CASSANDRA_MAPPINGS =
       CassandraMappings.builder()
-          .put("ASCII", UnifiedMappingProvider.Type.STRING, Row::getString, valuePassThrough)
-          .put("BIGINT", UnifiedMappingProvider.Type.LONG, Row::getLong, valuePassThrough)
-          .put("BLOB", UnifiedMappingProvider.Type.STRING, Row::getBytes, ByteBufferToHexString)
-          .put("BOOLEAN", UnifiedMappingProvider.Type.BOOLEAN, Row::getBool, valuePassThrough)
-          .put("COUNTER", UnifiedMappingProvider.Type.LONG, Row::getLong, valuePassThrough)
-          .put("DATE", UnifiedMappingProvider.Type.DATE, Row::getDate, localDateToAvroLogicalDate)
+          .put(
```

Comment:
Summary on the diff:
For querying `list<Type>` and `Set<Type>`,  we need to pass the type's class to the mapping layer. 
Adding the class has resulted in the teleport repo's formatter to split each put into multi lines.